examples/tao.rs

```diff
@@ -38,12 +46,10 @@ fn main() {
 
     event_loop.run(move |event, _, control_flow| {
         // We add delay of 16 ms (60fps) to event_loop to reduce cpu load.
-        // This can be removed to allow ControlFlow::Poll to poll on each cpu cycle
         // Alternatively, you can set ControlFlow::Wait or use TrayIconEvent::set_event_handler,
         // see https://github.com/tauri-apps/tray-icon/issues/83#issuecomment-1697773065
-        *control_flow = ControlFlow::WaitUntil(
-            std::time::Instant::now() + std::time::Duration::from_millis(16),
-        );
+        *control_flow = ControlFlow::Poll;
+        std::thread::sleep(std::time::Duration::from_millis(16));
```

Comment:
Because otherwise the event loop won't wake up after the timeout, only when the user clicks the tray icon.

https://github.com/tauri-apps/tao/issues/988